frappe/database.py

```diff
@@ -741,6 +741,7 @@ def commit(self):
 		frappe.local.rollback_observers = []
 		self.flush_realtime_log()
 		self.enqueue_global_search()
```

Comment:
also add global search to this?

Reply:
Same pull? 
I'll look into it.

Reply:
@rmehta 
`self.enqueue_global_search()` will be replaced with `execute_enqueued_jobs()`

from where should I call following?
```
try:
	frappe.enqueue('frappe.utils.global_search.sync_global_search',
		now=frappe.flags.in_test or frappe.flags.in_install or frappe.flags.in_migrate,
		flags=frappe.flags.update_global_search, enqueue_after_commit=True)
except redis.exceptions.ConnectionError:
	sync_global_search()
```

Reply:
Check from where it is currently called, both should follow the sames system